force-app/main/default/lwc/wireGetRecord/wireGetRecords.js

```diff
@@ -0,0 +1,22 @@
+import { LightningElement, wire, api } from 'lwc';
```

Comment:
Is this supposed to be a separate component? Please create the component in its own folder.